src/test/java/examples/MyCredentials.java

```diff
@@ -6,8 +6,20 @@
 
 public class MyCredentials {
 
-    public static String WEBMATE_SELENIUM_URL = "https://app.webmate.io:44444/wd/hub";
-    public static String MY_WEBMATE_USERNAME = "xxxx@xxxxxxxxx.com";
-    public static String MY_WEBMATE_APIKEY = "xxxxxx-xxxxx-xxxx-ba43-da86b97734eb";
-    public static ProjectId MY_WEBMATE_PROJECTID = new ProjectId(UUID.fromString("xxxxxx-1b49-4eb0-bb3a-xxxxxxxxx"));
+    /**
+     * These are the credentials with which you can use the SDK with the demo instance of Webmate (https://demo.webmate.io).
+     * Simply fill in the username, api key and project ID
+     * Details on how to get your API Key and Project ID can be found in the official webmate documentation.
+     *         https://docs.webmate.io/reference/generate-api-key/?searchTerm=api
+     * There is no need to change the Selenium URL or the Webmate API URI
+     */
+    public static String WEBMATE_SELENIUM_URL = "https://selenium-demo.webmate.io/wd/hub";
```

Comment:
app.webmate.io should still be the default, but leave the settings for demo to be commented in when needed